config/plugins.js

```diff
@@ -13,7 +13,7 @@ module.exports = ({env}) => ({
     sentry: {
       enabled: true,
       config: {
-        dsn: 'https://1b4a30f38ea04b9b956481f226a83e4e@o1123967.ingest.sentry.io/6372271',
+        dsn: 'https://9b74d2bd2a50432da68982c3c4d72420@o4505235023921152.ingest.sentry.io/4505235032899584',
```

Comment:
Can we read is from environment? That way we can inject the dsn as a secret. See `jwtSecret` for an example.